Cargo.toml

```diff
@@ -87,6 +88,8 @@ winapi = { version = "0.3.4", features = ["winsock2", "winuser", "shellapi"] }
 daemonize = "0.3"
 
 [features]
+default = ["accounts"]
+accounts = ["ethcore-accounts", "ethcore-secretstore/accounts", "parity-rpc/accounts"]
```

Comment:
This causes Parity to be built with "secretstore" feature enabled by default (since default features are "accounts" and it implies "ethcore-secretstore/accounts"). I guess we need something like `accounts AND secretstore = ["ethcore-secretstore/accounts"]`, but I'm not sure it could be defined within Cargo.toml. Probably replace `accounts => [..., "ethcore-secretstore/accounts", ...]` with `secretstore => [""ethcore-secretstore/accounts", "ethcore-accounts"]`?

Reply:
or even better - just remove "ethcore-secretstore/accounts" from "accounts" - so that there could be `--features ""`, `--features "accounts"`, `--features "secretstore"` and `--features "secretstore accounts"`.

Reply:
I don't think that would work, we need to enable `ethcore-secretstore/accounts` if both `secretstore` and `accounts` features are enabled. I changed it so that we require `accounts` in case we compile with `secretstore`